src/test/java/br/com/alura/comex/relatorios/RelatorioVendasPorCategoriaTest.java

```diff
@@ -0,0 +1,71 @@
+package br.com.alura.comex.relatorios;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import br.com.alura.comex.Pedido;
+import br.com.alura.comex.relatorios.RelatorioVendasPorCategoria.VendasCategoria;
+
+class RelatorioVendasCategoriaTest {
+    List<Pedido> pedidos = new ArrayList<>();
+    RelatorioVendasPorCategoria relatorioVendasCategoria = new RelatorioVendasPorCategoria();
+    private Consumer iniciarTeste() {
+        Consumer consumer = Mockito.mock(Consumer.class);
+        relatorioVendasCategoria.imprimirRelatorio(pedidos, consumer);
+        return consumer;
+    }
+
+    @Test
+    void gerarRelatorioComApenasUmPedido() {
+        Pedido pedido = new Pedido(
+                "JOGOS", "The Last Of Us 2", "Cassia", new BigDecimal("100.00"), 1,
+                LocalDate.of(2022, 6, 1)
+        );
+
+        pedidos = List.of(pedido);
```

Comment:
Aqui, poderia declarar a variável diretamente!

```suggestion
       List<Pedido> pedidos = List.of(pedido);
```